.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Release skupper-router
+#on:
+#  release:
+#    types: [published]
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build and publish skupper-router image
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
@ganeshmurthy one thing that might be interesting to evaluate would be the
use of create-release action, see: [](https://github.com/actions/create-release)

This manual task works ok, from the POV of releasing images, but there won't be
any release related information on the repository.

Reply:
> @ganeshmurthy one thing that might be interesting to evaluate would be the use of create-release action, see: https://github.com/actions/create-release
> 
> This manual task works ok, from the POV of releasing images, but there won't be any release related information on the repository.

Yes, Please see my earlier comment in this PR. I tried using the on: release which did not work. @jiridanek suggested a fix to my approach but then we agreed to keep this process manual.